src/pages/Wallet/WalletHeader/WalletHeader.js

```diff
@@ -0,0 +1,26 @@
+import WalletIcon from "@mui/icons-material/Wallet";
+import {
+  ContentContainer,
+  InnerCircle,
+  PendingTransfers,
+  Title,
+  TitleContainer,
+} from "./WalletHeaderStyled";
+
+const WalletHeader = ({ title }) => {
+  return (
+    <>
+      <ContentContainer>
+        <InnerCircle>
+          <WalletIcon style={{ color: "#fff", fontSize: 70 }} />
+        </InnerCircle>
+        <TitleContainer>
+          <Title>{title}</Title>
+          <PendingTransfers>Pending Transfers - 4</PendingTransfers>
```

Comment:
Number of transfers should not be hardcoded - we can take it from the API {{baseUrl}}/transfers?state=pending